tests/integration/providers/ldap/delta_inbound_sync_tests.py

```diff
@@ -583,7 +581,32 @@ def test_create_fake_user(ldap_connection, user):
     time.sleep(1)
     email = "%s@clouddev.corporate.t-mobile.com" % user["common_name"]
     result = is_user_in_db(email)
+    syncflag_fetched = get_syncmatch_count(user["common_name"])
     assert result is True
+    assert syncflag_fetched is True
+
+
+def get_syncmatch_count(username_provided):
+    """
+    Function that results TRUE or False,
```

Comment:
the docstring is a bit confusing, I'll add it to our docs today, but we're trying to loosely follow this docstring styleguide, again, nitpicky and not a huge deal. http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

Reply:
updating it correctly